proto/mezo/bridge/v1/bridge.proto

```diff
@@ -40,10 +40,33 @@ message AssetsLockedEvent {
   string token = 4;
 }
 
-// ERC20TokenMapping defines a mapping between an ERC20 token on the source chain
-// and on the Mezo chain.
+// AssetsUnlockedEvent represents the event where inbound assets are released
+// from the bridge.
+message AssetsUnlockedEvent {
```

Comment:
Let's please reorder fields in the same way as in `IAssetsBridge`. This will be a help for readers. Remember about updating usage places (`SaveAssetsUnlocked`).

Reply:
4bf631d5